modules/compiler/builtins/CMakeLists.txt

```diff
@@ -35,9 +35,13 @@ include(cmake/FindBuiltinsTools.cmake)
 if(NOT CA_EXTERNAL_BUILTINS AND CA_RUNTIME_COMPILER_ENABLED)
   if(NOT DEFINED CA_BUILTINS_TOOLS_DIR)
     if(CMAKE_CROSSCOMPILING)
-      message(FATAL_ERROR
-        "Cross compiled builds requires CA_EXTERNAL_BUILTINS or "
-        "CA_BUILTINS_TOOLS_DIR to be set!")
+      if(EXISTS ${LLVM_NATIVE_TOOL_DIR})
```

Comment:
Just making sure, what does this do when `LLVM_NATIVE_TOOL_DIR` is not defined? Is `if(EXISTS )` well-defined, or do we need something like `if(DEFINED LLVM_NATIVE_TOOL_DIR AND EXISTS "${LLVM_NATIVE_TOOL_DIR}")` ?

Reply:
The [documentation](https://cmake.org/cmake/help/latest/command/if.html#exists) states that `if(EXISTS ` is false if it's given an empty string, and I think an undefined variable evaluates to an empty string, so this looks ok to me, but I can add the check if you think it's useful

Reply:
That's fine then, thanks for checking.